certbot/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * Make Certbot snap find externally snapped plugins
 * Function `certbot.compat.filesystem.umask` is a drop-in replacement for `os.umask`
   implementing umask for both UNIX and Windows systems.
+* Merge the snap-plugin branch into master. This includes a snapcraft.yaml for
```

Comment:
nit: Since this isn't a change that affects any of our distribution methods, I personally don't think we should have a changelog entry about this. I think our changelog is mostly for our users and packagers and this isn't useful information for them.

We can keep it if you disagree for some reason though.

Reply:
sgtm

Reply:
this may disappear when I force-push, but: done.